swagger.yaml

```diff
@@ -1,42 +1,244 @@
 swagger: "2.0"
+
 info:
   title: dp-search-reindex-api
   description: "Provides details about search reindex jobs and enables creation and running of them"
   version: 1.0.0
   license:
     name: "Open Government Licence v3.0"
     url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+
 basePath: "/v1"
+
 schemes:
   - http
+
 tags:
-  - name: "hello"
+  - name: "Public"
+
 paths:
-  /hello:
+
+  /jobs/{id}:
     get:
       tags:
-        - hello
-      summary: Example hello world endpoint
-      description: Returns an example payload.
+        - "Public"
+      summary: Get a specific job.
+      description: Get the specific search reindex job that has the id given in the path.
+      parameters:
+        - $ref: '#/parameters/id'
       produces:
         - application/json
       responses:
         200:
           description: OK
```

Comment:
It would be better to say something along the lines of "Successfully returned a job for a given id" instead of just an OK

Reply:
I concur

Reply:
done